app/api/helpers/permissions.py

```diff
@@ -227,8 +227,8 @@ def decorated_function(*args, **kwargs):
         if user.is_staff:
             return f(*args, **kwargs)
         if 'event_id' in kwargs and (
-                    user.is_moderator(kwargs['event_id']) or
-                    user.has_event_access(kwargs['event_id'])):
+            user.is_moderator(kwargs['event_id']) or
+                user.has_event_access(kwargs['event_id'])):
```

Comment:
Revert changes in this file. And create a decorator in this file similar to others and apply that to the functions

Reply:
@iamareebjamal But these are the simple indentation corrections, I don't see the point of reverting those changes.

Reply:
Formatting changes should be done in separate PRs. Someone will be trying to debug something and see the blame and blame you for modufying it in a PR for fixing permissions whereas you only indented files